docs/pages/kubernetes-access/guides/discovery.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Kubernetes Discovery Guides (Preview)
+description: Detailed guides for configuring Kubernetes Auto-discovery.
```

Comment:
Should we say "Kubernetes Auto-Discovery" everywhere, rather than "Kubernetes Discovery"?